libs/barista-components/switch/tsconfig.json

```diff
@@ -1,5 +1,5 @@
 {
-  "extends": "../tsconfig.json",
+  "extends": "../../../tsconfig.base.json",
```

Comment:
please extend from the `libs/barista-components/tsconfig.json`.

Reply:
Thanks for catching this.